crates/fhir-sdk/Cargo.toml

```diff
@@ -36,22 +36,22 @@ r5 = ["fhir-model/r5"]
 docs = []
 
 [dependencies]
-async-trait = {version = "0.1.68", optional = true}
+async-trait = {version = "0.1.79", optional = true}
 fhir-model = {path = "../fhir-model", version = "0.9.0", default-features = false}
-futures = {version = "0.3.28", optional = true}
-reqwest = {version = "0.11.16", features = ["json"], optional = true}
-serde = {version = "1.0.159", optional = true}
-serde_json = {version = "1.0.95", optional = true}
-thiserror = {version = "1.0.40", optional = true}
+futures = {version = "0.3.30", optional = true}
+reqwest = {version = "0.12.2", features = ["json"], optional = true}
```

Comment:
I think reqwest 0.12 needs the h2 feature as well enabled to support that right? Or is it enabled by default?

Reply:
Enabled by default